crates/nu-cmd-plugin/src/commands/plugin/use_.rs

```diff
@@ -0,0 +1,81 @@
+use nu_engine::command_prelude::*;
+
+#[derive(Clone)]
+pub struct PluginUse;
+
+impl Command for PluginUse {
+    fn name(&self) -> &str {
+        "plugin use"
+    }
+
+    fn usage(&self) -> &str {
+        "Load a plugin from the plugin cache file into scope."
+    }
+
+    fn signature(&self) -> nu_protocol::Signature {
+        Signature::build(self.name())
+            .input_output_types(vec![(Type::Nothing, Type::Nothing)])
+            .named(
+                "plugin-config",
+                SyntaxShape::Filepath,
+                "Use a plugin cache file other than the one set in `$nu.plugin-path`",
+                None,
+            )
+            .required(
+                "name",
+                SyntaxShape::String,
+                "The name of the plugin to load (not the filename)",
+            )
+            .category(Category::Plugin)
+    }
+
+    fn extra_usage(&self) -> &str {
+        r#"
+This command is a parser keyword. For details, check:
+  https://www.nushell.sh/book/thinking_in_nu.html
+
+The plugin definition must be available in the plugin cache file at parse time.
+Run `plugin add` first in the REPL to do this, or from a script consider
+preparing a plugin cache file and passing `--plugin-config`, or using the
+`--plugin` option to `nu` instead.
+
+If the plugin was already loaded, this will reload the latest definition from
+the cache file into scope.
+"#
+        .trim()
+    }
+
+    fn search_terms(&self) -> Vec<&str> {
+        vec!["add", "plugin", "register", "scope", "use"]
```

Comment:
The unwritten rule here is to not have any search terms here that are in the name of the command, because these are exclusively here for `help -f`, searching. I've noticed this on a few of your recent commands and forgot to mention it. So, what I'm saying, to be clear, this should probably remove `plugin` and `use` in the search terms here because they're part of the command name.

Reply:
I added the test that is draconian about this back in the day when all commands just lived in `nu-command` local to that crate (https://github.com/nushell/nushell/pull/6380).
It wasn't moved to a better place during cratification, so it doesn't affect new additions of `nu-cmd-*` crates.

Reply:
Thanks - let me know if I step on anything like this in the future too, for sure